### PR TITLE
Add win2016 and win2016nano series

### PR DESCRIPTION
**Pull request by mhilton**
Also consolidated all the lists of series into one place to make it
easier to update in future.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1299/
Test PASSed.

File: internal/charmstore/search.go [hunk 1]
```diff
@@ -134,7 +119,7 @@ func (s *Store) UpdateSearchBaseURL(baseURL *charm.URL) error {
 	// (2.4) would require every field to be enumerated in this query, which
 	// would make it too fragile.
 	iter := s.DB.Entities().Pipe([]bson.D{
-		{{"$match", bson.D{{"baseurl", baseURL}}}},
+		{{"$match", bson.D{{"baseurl", baseURL}, {"development", false}}}},
```
Comment (frankban) [inline internal/charmstore/search.go:122]: Good catch.

File: internal/charmstore/search.go [hunk 2]
```diff
@@ -86,7 +71,7 @@ func (s *Store) UpdateSearch(r *router.ResolvedURL) error {
 	if s.ES == nil || s.ES.Database == nil {
 		return nil
 	}
-	if deprecatedSeries[r.URL.Series] || r.Development {
+	if r.Development || r.URL.Series != "" && !series.Series[r.URL.Series].SearchIndex {
```
Comment (frankban) [inline internal/charmstore/search.go:74]: Maybe I am missing something but shouldn't this be `|| r.URL.Series == ""`?
If so, how are the tests passing?

Comment (frankban) [inline internal/charmstore/search.go:74]: Oh I see, nevermind, it || and then &&.

File: internal/series/series.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+const (
+	Ubuntu  Distribution = "ubuntu"
+	CentOS  Distribution = "centos"
+	Windows Distribution = "windows"
+)
+
+// SeriesInfo contains the information the charmstore knows about a
+// series name.
+type SeriesInfo struct {
```
Comment (frankban) [inline internal/series/series.go:20]: Nice!

**Comment by frankban** [comment]
:+1: ship it!

**Comment by urosj** [comment]
:+1:

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**